k8sfv/run-test

```diff
@@ -211,13 +211,27 @@ if test -n "${GINKGO_FOCUS}"; then
     docker exec ${prefix}-felix /bin/sh -c \
 	   "${run} -ginkgo.focus \"${GINKGO_FOCUS}\""
 elif ${JUST_A_MINUTE}; then
+    tmp=$(mktemp)
     # Run all the tests that are _not_ marked as "[slow]", and fail
     # the test if any of those takes longer than 2 minutes.  A k8sfv
     # test should be marked as "[slow]" if it normally takes longer
     # than 1 minute - so this allows a buffer of 1 extra minute to
     # allow for executor slowness.
     docker exec ${prefix}-felix /bin/sh -c \
-	   "${run} -ginkgo.skip \"\\[slow\\]\" -ginkgo.slowSpecThreshold 120" | perl -pe 'END { exit $status } $status=1 if /SLOW TEST|FAIL!/;'
+       "${run} -ginkgo.skip \"\\[slow\\]\" -ginkgo.slowSpecThreshold 120" | tee $tmp
+	# Fail if the docker command returned failure
+	test ${PIPESTATUS[0]} -eq 0
```

Comment:
Does this definitely fail the whole script?  ISTR that a bit of care is needed inside an `if` block but I could be wrong.

Reply:
I just tried this test script
```
#!/bin/bash -ex

x=1
if [ $x -eq 0 ]; then
    echo "x is 0"
else
    echo "Here"
    test 1 -eq 0
    echo "There"
fi
```
and got this output
```
> ./test.sh 
+ x=1
+ '[' 1 -eq 0 ']'
+ echo Here
Here
+ test 1 -eq 0
```
I am going to go ahead and change it though because I think a simple if statement with a message when it exits will be better.